pkg/api/service/util.go

```diff
@@ -53,16 +52,8 @@ func GetLoadBalancerSourceRanges(service *api.Service) (netsets.IPNet, error) {
 			return nil, fmt.Errorf("service.Spec.LoadBalancerSourceRanges: %v is not valid. Expecting a list of IP ranges. For example, 10.0.0.0/24. Error msg: %v", specs, err)
 		}
 	} else {
-		val := service.Annotations[api.AnnotationLoadBalancerSourceRangesKey]
-		val = strings.TrimSpace(val)
-		if val == "" {
-			val = defaultLoadBalancerSourceRanges
-		}
-		specs := strings.Split(val, ",")
-		ipnets, err = netsets.ParseIPNets(specs...)
-		if err != nil {
-			return nil, fmt.Errorf("%s: %s is not valid. Expecting a comma-separated list of source IP ranges. For example, 10.0.0.0/24,192.168.2.0/24", api.AnnotationLoadBalancerSourceRangesKey, val)
-		}
+		specs := strings.Split(defaultLoadBalancerSourceRanges, ",")
+		ipnets, _ = netsets.ParseIPNets(specs...)
```

Comment:
Should I return error here?
I think there is no error when parse defaultLoadBalancerSourceRanges("0.0.0.0/0").